### DISCUSSION TITLE
有什么在java中可以直接调用的保存文章的方法吗？

### DISCUSSION THREAD

**Discussion by three-ant** [halo-dev/halo#7148] [Q&A]
我想编写一个插件来创建文章，想在java中调用接口方法，但是没有找到。只找到了以下前端接口。
api.console.halo.run/v1alpha1/posts
有什么在java中可以直接调用的方法吗？

**Comment by JohnNiang** [comment]
Hi @three-ant , thank you for reaching out here!

目前 Halo core 暂时没有暴露任何文章服务接口，所以无法实现这个需求。

作为 Workaround：你可以尝试手动复制[对应的方法](https://github.com/halo-dev/halo/blob/156a30496c3d3e2f446a0987248ff12f6ecfc4f0/application/src/main/java/run/halo/app/content/impl/PostServiceImpl.java#L189)临时解决这个问题。

**Reply by three-ant** [reply]
好的，谢谢
